Resources/Private/fluid_styled_content/Templates/Mannotation.html

```
@@ -103,12 +103,14 @@ <h3 class="ce-headline-{data.header_position}" id="c{data.uid}">
 
         <f:for each="{points.{img-i.index}}" as="point" iteration="i">
-            <f:if condition="{point.description}">
+            <f:if condition="{point.title} || {point.description}">
                 <div class="focuspoint__description" id="focuspoint-{data.uid}-{i.cycle}">
                     <strong>{f:if(condition: '{point.type}==marking', then: '{i.cycle}. ')}{point.title}</strong>
-                    <p>
-                        <f:format.nl2br>{point.description}</f:format.nl2br>
-                    </p>
+                    <f:if condition="{point.description}">
+                        <p>
+                            <f:format.nl2br>{point.description}</f:format.nl2br>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)